api/models/event.js

```diff
@@ -12,6 +12,7 @@ module.exports = (sequelize, DataTypes) => {
     SOCKET_IO: 'socket.io',
     SITE_USER: 'site-user',
     REQUEST_HANDLER: 'request-handler',
+    USER_ACTION: 'user-action',
```

Comment:
should we differentiate between a user and an operator?

Reply:
great suggestion 👍🏿  ... i've added ADMIN_ACTION: 'admin-action'